docs/installing/installing-gke.rst

```diff
@@ -100,7 +100,7 @@ Create and Deploy tsuru-dashboard app:
 
 ::
 
-   $ tsuru app create dashboard
+   $ tsuru app create dashboard --team
```

Comment:
this is not accurate, if the user runs this exact command tsuru will ask the user to specify a valid team